src/steps/create-pictures.js

```diff
@@ -77,20 +84,22 @@ function isMediaImage(node) {
 export default async function createPictures({ content }) {
   const { hast } = content;
 
-  visitParents(hast, isMediaImage, (img, parents) => {
+  visitParents(hast, (node) => node.tagName === 'img', (img, parents) => {
     const { src, alt, title } = img.properties;
```

Comment:
`properties` can be undefined:

```suggestion
    const { src, alt, title } = img.properties || {};
```